cpp/src/arrow/compute/kernels/scalar_string.cc

```diff
@@ -4028,17 +4087,14 @@ const FunctionDoc utf8_lower_doc(
     "Transform input to lowercase",
     ("For each string in `strings`, return a lowercase version."), {"strings"});
 
-const FunctionDoc ascii_reverse_doc(
-    "Reverse ASCII input",
-    ("For each ASCII string in `strings`, return a reversed version.\n\n"
-     "This function assumes the input is fully ASCII.  If it may contain\n"
-     "non-ASCII characters, use \"utf8_reverse\" instead."),
-    {"strings"});
+const FunctionDoc utf8_capitalize_doc(
+    "Capitalize the first codepoint of UTF8 input",
+    ("For each UTF8 string in `strings`, return a capitalized version."), {"strings"});
```

Comment:
Nit, but no need to specify "UTF8". The input is of string type, so UTF8 by definition.